projects/batfish-common-protocol/src/test/java/org/batfish/datamodel/StaticRouteTest.java

```diff
@@ -0,0 +1,93 @@
+package org.batfish.datamodel;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+
+import java.io.IOException;
+import org.batfish.common.BatfishException;
+import org.batfish.common.util.BatfishObjectMapper;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+/** Tests for {@link StaticRoute} */
+public class StaticRouteTest {
+
+  @Rule public ExpectedException _thrown = ExpectedException.none();
+  StaticRoute.Builder _srBuilder;
+
+  @Test
+  public void checkAllAttrs() {
+    _srBuilder
+        .setNextHopIp(new Ip("192.168.1.1"))
+        .setNetwork(Prefix.ZERO)
+        .setNextHopInterface("192.168.1.2")
+        .setAdministrativeCost(1)
+        .setTag(0);
+    StaticRoute sr = _srBuilder.build();
+    assertThat(sr.getNextHopIp(), is(new Ip("192.168.1.1")));
+    assertThat(sr.getNetwork(), is(Prefix.ZERO));
+    assertThat(sr.getNextHopInterface(), is("192.168.1.2"));
+    assertThat(sr.getAdministrativeCost(), is(1));
+    assertThat(sr.getTag(), is(0));
+  }
+
+  @Test
+  public void checkNullNextHop() {
+    _srBuilder.setNetwork(Prefix.ZERO).setNextHopIp(null);
```

Comment:
no, it does not crashes, I just wanted to check the value assigned to nextHopIp when it is set to Null